### PR TITLE
end-to-end taxi example

### PR DESCRIPTION
**Pull request by rikturr**
This takes the end-to-end taxi example from `examples-cpu/nyc-taxi` and creates a dedicated quickstart for it. Most content was unchanged

### CODE REVIEW

**Review by jsignell — CHANGES_REQUESTED** [review]
I suspect this will need `hvplot` if there is a dashboard notebook. You can use the `examples-cpu` image or include a start script like the standalone `dashboard` quickstart does. 

Also is end-to-end the best name? Maybe data-sciece-pipeline is better?

**Comment by jsignell** [comment]
running integration tests: https://github.com/saturncloud/release-images/runs/2124696802?check_suite_focus=true

File: examples/data-science-pipeline/.saturn/saturn.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "image": "saturncloud/saturn:2021.02.22",
```
Comment (rikturr) [inline examples/data-science-pipeline/.saturn/saturn.json]:
```suggestion
    "image": "saturncloud/examples-cpu:2021.02.22",
```

**Comment by rikturr** [comment]
renamed to data-science-pipeline, switched image to examples-cpu, and integration test passes 🎉 
https://github.com/saturncloud/release-images/runs/2134316216?check_suite_focus=true

**Review by jsignell — APPROVED** [review]

**Review by jnolis — APPROVED** [review]
This one looks good to me! Since we don't plan to make it a doc yet you don't necessarily have to conform to the rules like "top cell is just the title", however if it's easy to do that before you merge you might as well make the changes I listed for the other repos.

**Pull request merged**